elrond-wasm-derive/src/preprocessing/substitution_algorithm.rs

```diff
@@ -41,5 +40,5 @@ pub(super) fn perform_substitutions(
             break;
         }
     }
-    proc_macro::TokenStream::from_iter(result.into_iter())
+    result.into_iter().collect::<proc_macro::TokenStream>()
```

Comment:
```suggestion
    result.into_iter().collect()
```
The type can be inferred from the function return type.